src/main/resources/DocumentDistributionMainProcess.xsd

```diff
@@ -44,14 +44,17 @@
 	</element>
         	
         <element name="DocumentDistributionMainProcessProcessResponse">
-		<complexType>
-			<sequence>
-				<element name="jobId" type="string"/>
-                <element name="statusCode" type="string"/>
-                <element name="statusMessage" type="string"/>
-			</sequence>
-		</complexType>
-	</element>
+			<complexType>
+				<sequence>
+					<element name="jobId" type="string"/>
+	                <element name="statusCode" type="string"/>
```

Comment:
The diff says that lines 50-51 are indented at a different level here compared to <----. Is that supposed to be the case?

Reply:
yeah, I just fixed the indentation on a couple lines.  The main change is lines 56-57